auth-agents-common/src/main/java/org/apache/atlas/policytransformer/CachePolicyTransformerImpl.java

```diff
@@ -176,6 +177,8 @@ public ServicePolicies getPoliciesDelta(String serviceName, Map<String, EntityAu
 
                 List<AtlasEntityHeader> atlasServicePolicies = allAtlasPolicies.stream().filter(x -> serviceName.equals(x.getAttribute(ATTR_POLICY_SERVICE_NAME))).collect(Collectors.toList());
                 List<RangerPolicyDelta> policiesDelta = getRangerPolicyDelta(service, policyChanges, atlasServicePolicies);
+                Map<String, RangerPolicyDelta> deletedPolicyDeltas = getRangerPolicyDeleteDelta(service, policyChanges);
+                servicePolicies.setDeleteDeltas(deletedPolicyDeltas);
```

Comment:
If we handle delete on apply delta side, it would have same iteration and keep the delta creation logic independent of how apply is working. Also, mapping update event to CREATE delta above might become confusing in future for someone else.